src/api.rs

```diff
@@ -62,7 +62,7 @@ impl Client {
     }
 
     /// Send the constructed URL to the newsapi server
-    pub fn send<T>(&self) -> Result<T, NewsApiError>
+    pub async fn send<T>(&self) -> Result<T, NewsApiError>
```

Comment:
Maybe we should make the methods explicit, if we're changing send to be async why not go with send_sync and send_async and do the same for the other methods?

Reply:
Personally, I think async is more interesting for the future of the lib since it is an API wrapper.
Async is a big jump and probably a good reason for a breaking change since in the future most users will be expecting async.